components/Labels/FilterLabelBox.tsx

```diff
@@ -21,17 +22,32 @@ export default function FilterLabelBox(props: {
   } = useQuery(["labels", searchText], () => getLabels(searchText));
 
   return (
-    <div className={styles.box}>
-      <TopSection handleClose={props.handleClose} />
-      <SearchSection setSearchText={setSearchText} />
-      <LabelSection labels={labels} isLoading={isLoading} error={error} />
+    <div
+      style={{
+        height: "100vh",
```

Comment:
Please use `module.scss` file to specify styling to have the components as clean as possible.